abs.yaml

```diff
@@ -1,3 +1,3 @@
 build_parameters:
   - "--suppress-variables"
-  - "--error-overlinking"
+  - "--no-error-overlinking"
```

Comment:
Prefer using `missing_dso_whitelist` instead. This has the advantage to selectively disable this check rather than lowering every issues to warnings.